### ISSUE TITLE
Notes, and a new timeline, from the AAS editorial get-together

### ISSUE THREAD

**Issue by drphilmarshall** [LSSTScienceCollaborations/ObservingStrategy#264]
Dear @LSSTScienceCollaborations/observing-strategy-white-paper-editors and @LSSTScienceCollaborations/observing-strategy-white-paper-contributors ,

A few of us met up yesterday here at AAS227 as planned ( #235 ). We discussed the white paper, it's current status following your recent efforts (thank you! :-) and what we need to do next and when. Below is a maximally concise summary! Please read, your comments are very welcome, as always :-)

Cheers

Phil

* While the white paper continues to come together well (with notable improvement in homogeneity of structure and some great new content), it is still some way from being ready for review.
* The primary missing pieces in the majority of science sections are 1) the equations/pseudocode and description of Figures of Merit, 2) the diagnostics and summaries that these will depend on, and 3) the brief description of the research program to define and compute the metrics in 1) and 2). In some cases this is because a year-scale research project is needed to figure things out, but in other cases there are approximations that could be made on a much shorter timescale.
* Beth suggested that our next internal target should be for us to produce the missing pieces  listed above, so that the Sims team can assess them and advise on turning them into functioning MAF code.
* We discussed reasonable time frames for approximate metric development ("a few months"), and external time frames to fit in with. The August project and community meeting in Tucson would be a good time to get together and look at the MAF analysis results on a set of new, "Tier 2", OpSim runs, which are due to be completed in early summer (and include rolling cadence, WFIRST coverage and other exciting new things). Our goal should be to have the white paper version 1 done *before* the Tier 2 runs are released, so that we will have adhered to the principle of first evaluating before optimizing. This suggests an *early summer deadline for the science metric analysis of enigma1189 and friends.*
* We recalled that LSST publications board review is required before we post to the arxiv, and that this will take several weeks. We then agreed that a realistic deadline for submission for LSST review could be May 31, with arxiv submission following on (hopefully) June 30. 
* Internally, we should then aim to have a final draft ready for editorial review by April 30. To give the Sims team a month to assess and potentially implement MAF metrics based on equations and pseudocode, the latter would need to be ready by February 28, to allow a month of MAF implementation and a month of science team experimentation, analysis, and write-up.

Resulting Timeline:

**1/31**
**2/28:**  Finish equations/pseudocode and description of metrics, as far as possible.
**3/31**
**4/30:**  Final draft of white paper, for internal editorial review
**5/31:**  Submission for LSST Pub Board Review
**6/30:**  Submission to arxiv

This gives us January and February to design our metrics (both diagnostic and FoM), and then March and April to work with the Sims team on implementing and applying them to the Tier 1 sims. What do you think?

To-do list arising from the meeting:

* [x] Phil to convert timeline to milestones, following discussion
* [x] Phil to work on the introduction text, so as to make the definitions of FoMs, diagnostics, and metrics clear to everyone
* [ ] Phil to email LSST project and science list, DDF authors, and LSST e-News, once introduction is improved, to let everyone know the plan (in case they want to come and join in). 
* [x] Peter to collect list of all available metrics in an appendix, to show contributors what is available. Ideally this will be hyperlinked to the functions themselves; other hyperlinks to documentation should also be included in the appendix.

Am I missing anything? Thanks!

**Comment by drphilmarshall** [comment]
Everyone I talked to at AAS was down with this new timeline. I'll just get to work on my action items this week unless I hear from you!

**Comment by drphilmarshall** [comment]
TL;DR: Please go through your issues, closing whatever you can, and reply to this message with any questions you have about the white paper! Back in January we said we'd have the Final draft (for internal review) done by April 28.


Dear LSST Observing Strategy White Paper Editors and Contributors,

After a long winter of thinking about your metrics and passing equations and pseudocode to the MAF team, it is now April and we are less than a month away from the Final Draft deadline we set ourselves back in January (as you can see, on our [milestones](https://github.com/LSSTScienceCollaborations/ObservingStrategy/milestones)).  Hopefully between now and then some of you will be able to use the new metrics that the MAF team have assembled to make plots for your science case sections! If we manage to stay on schedule, on April 28 we would start reviewing each other's sections.

To get a sense of how far you and your teams will be able to get by the end of this month, please reconnect with your issues and be aggressive in closing them out. In many cases I suspect the work has been done, but the to-do list item has not been ticked off, so to speak. You can find the issues for your chapter by filtering the list by label (eg, here are the [Variables and Transients issues](https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues?q=is%3Aissue+is%3Aopen+label%3A%22Variables+and+Transients%22)), or by assignee (eg, here are the issues assigned to [drphilmarshall](https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues?q=assignee%3Adrphilmarshall+is%3Aopen). Most things you see on the [issue list](https://github.com/LSSTScienceCollaborations/ObservingStrategy/issues) are hyperlinked so that when you click on them, you get a filtered list (I clicked on my photo to get my assigned issues, for example).

If we put some concerted effort into closing issues wherever possible, this should make the issues system much more useful as a tool for seeing what we still have to do. As you do this, you may find yourselves needing to make new issues. That's good - whether or not you have metrics to use, your sections can still be finished by the deadline. (They just won't have much quantitative to say, yet.)  Editors: Don't forget to label, milestone and assign your new issues as you go.

Meanwhile, please do just reply to this thread with any questions you have about the timeline, what is needed, and so on!

Cheers

Phil
